woocommerce/woocommerce-ios
# [Mobile Payments] Add networking for capturing WCPay payment intents (and updating order status)

Closes #3970 

Changes:
- Adds a new networking request for `captureOrderPayment` to `WCPayRemote`
- `captureOrderPayment` will send a request through Jetpack to the self-hosted site to instruct the site to capture the payment intent created by the Stripe Terminal SDK and update the order to reflect payment completion
- Note that the `payments/orders/\(sampleOrderID)/capture` endpoint does not yet exist in the WooCommerce Payments plugin - see https://github.com/Automattic/woocommerce-payments/issues/1622

To Test:
- Since the endpoint doesn't exist yet, the appropriate test is to ensure all the new tests added to WCPayRemoteTests pass 

Update release notes:
